plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ResourceDetailsContentProvider.java

```diff
@@ -90,6 +90,21 @@ private void getDeploymentConfigChildren(Collection<ResourceProperty> properties
 
 	private void getBuildConfigChildren(Collection<ResourceProperty> properties, IBuildConfig config) {
 		IBuildStrategy buildStrategy = config.getBuildStrategy();
+		addStrategyTypeProperties(properties, buildStrategy);
+		properties.add(new ResourceProperty("source URL", config.getSourceURI()));
+		properties.add(new ResourceProperty("output to", config.getOutputRepositoryName()));
+		List<String> triggers = config.getBuildTriggers().stream()
+				.map(trigger -> trigger.getType().toString())
+				.collect(Collectors.toList());
+		properties.add(new ResourceProperty("build triggers", triggers));
+	}
+
+	private void addStrategyTypeProperties(Collection<ResourceProperty> properties, IBuildStrategy buildStrategy) {
+		if (buildStrategy == null) {
+			properties.add(new ResourceProperty("strategy", "<UNKNOWN>"));
```

Comment:
the problem here is that there actually is something that^s defined, but it's unknown to the openshift-restclient-java, it cannot parse it, thus returning null. That's something different than "not provided". That's rather "unknown".